interfaces/builtin/opengl.go

```diff
@@ -172,6 +185,10 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 unix (send, receive) type=dgram peer=(addr="@var/run/nvidia-xdriver-*"),
 `
 
+type openGlInterface struct {
```

Comment:
```suggestion
type openglInterface struct {
```
to be consistent with rest of the names used in this file